### ISSUE TITLE
Failed when passing empty string to rules

### ISSUE THREAD

**Issue by chantouchsek** [chantouchsek/validator-js#1108]
```ts
export function isValidDate(dateString: any) {
  if (dateString === null || dateString === '') return false

  let testDate
  if (typeof dateString === 'number') {
    testDate = new Date(dateString)
    return typeof testDate === 'object'
  }
  testDate = new Date(dateString)
  if (testDate.toString() === 'Invalid Date')
    return false

  return checkFalsePositiveDates(dateString)
}
